tests/numberparsingcheck.cpp

```diff
@@ -87,7 +87,11 @@ void found_integer(int64_t result, const uint8_t *buf) {
   char *endptr;
   long long expected = strtoll((const char *)buf, &endptr, 10);
   if ((endptr == (const char *)buf) || (expected != result)) {
+#if (!(__MINGW32__) && !(__MINGW64__))
```

Comment:
I think we can avoid the ifdef by making a macro:

```c++
#if __MINGW32__ || __MINGW64__
#define SIMDJSON_LLD PRId64
#define SIMDJSON_LLU PRIu64
#else
#define SIMDJSON_LLD "lld"
#define SIMDJSON_LLU "llu"
#endif

fprintf(stderr, "Error: parsed %" SIMDJSON_LLD " out of %.32s, ", result, buf);
```

I'm fine with the 2 ifdefs we have now, but if you mess with common_defs for other reasons, this might be worth it :)

Reply:
Right. Both PRIu64 and PRId64 are part of the standard so I don't know why they don't work. But at some point, you run out of steam trying to find out why something that is perfectly fine by the standard does not work.

Reply:
Understandable :) I did find a stackoverflow indicating that you can get PRIu64 by including <inttypes.h>. But who knows if THAT exists in all the compilers we support ... and it's not worth spending more of your time right now.

Reply:
Oh. Right. I assumed we were importing these in the main library, but I think we do not. 

Thanks. (I knew of the header, I just assumed it was there already.)

Reply:
No. It is there, look in the files. We have cinttypes. It is another instance where mingw is busted.

Reply:
This flat out a bug on their part.